cocotb/drivers/xgmii.py

```diff
@@ -165,13 +166,20 @@ def layer1(packet):
                 struct.pack("<I", zlib.crc32(packet) & 0xFFFFFFFF))
 
     def idle(self):
-        """Helper to set bus to IDLE state"""
+        """
+        Helper function to set bus to IDLE state.
+        """
```

Comment:
It's actually appropriate to have the docstring as it previously was, one line with opening and beginning triple quotes on the same line, as long as it's not too long (which it does not appear to be).

Reply:
@kolojoe agreed, and that is consistent with PEP-257 as well.